docs/src/equations.md

```diff
@@ -10,8 +10,8 @@
 
 * ``\rho``: _mass_ in kg/m³. This is discretized at cell centers.
 * ``\boldsymbol{u}`` _velocity_, a vector in m/s. This is discretized via ``\boldsymbol{u} = \boldsymbol{u}_h + \boldsymbol{u}_v`` where
-  - ``\boldsymbol{u}_h = u_1 \boldsymbol{e}^1 + u_2 \boldsymbol{e}^2`` are the horizontal covariant part (with respect to the reference element), stored at cell centers.
-  - ``\boldsymbol{u}_v = u_3 \boldsymbol{e}^3`` is the vertical covariant part, stored at cell faces.
+  - ``\boldsymbol{u}_h = u_1 \boldsymbol{e}^1 + u_2 \boldsymbol{e}^2`` are the horizontal covariant vector (with respect to the reference element), stored at cell centers.
+  - ``\boldsymbol{u}_v = u_3 \boldsymbol{e}^3`` is the vertical covariant vector, stored at cell faces.
```

Comment:
I'm not a huge fan of the term "covariant vector": I think I would prefer "vector with horizontal covariant components" (but i also address this in #344.

Reply:
Okay, how do we want to proceed with these 2 somewhat overlapping PRs? Do you want me to update it here as you suggest, merge this, and then you can go on and rebase your branch?

Reply:
I applied your suggestion and pushed again